<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="urdf_export_assem#3.SLDASM">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="0.00134691271357942 -2.52657545412177E-05 0.472082902102997"
        rpy="0 0 0" />
      <mass
        value="13.3704893846402" />
      <inertia
        ixx="0.389862215162195"
        ixy="3.73442510141833E-05"
        ixz="0.00489552498258593"
        iyy="0.361904289007973"
        iyz="-0.000207680006355546"
        izz="0.413509038616086" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Right_wheel_link">
    <inertial>
      <origin
        xyz="-8.93952047617032E-06 0.0150000793737486 -1.16037505515365E-05"
        rpy="0 0 0" />
      <mass
        value="0.415938182180326" />
      <inertia
        ixx="0.000652914571865517"
        ixy="1.4107340831946E-10"
        ixz="2.68048423358216E-08"
        iyy="0.00125445432257594"
        iyz="-3.56322307486116E-12"
        izz="0.000652929076357562" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Right_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.796078431372549 0.796078431372549 0.796078431372549 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Right_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Right_wheel_Axis"
    type="continuous">
    <origin
      xyz="0 -0.212229686537745 0.0183338534091537"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Right_wheel_link" />
    <axis
      xyz="0 -1 0" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="Left_wheel_link">
    <inertial>
      <origin
        xyz="-1.36983615786788E-05 -0.0149999206509359 -5.18816478300987E-06"
        rpy="0 0 0" />
      <mass
        value="0.415938181435814" />
      <inertia
        ixx="0.000652942508895541"
        ixy="1.18132939340405E-10"
        ixz="1.85257141794357E-08"
        iyy="0.00125445432188265"
        iyz="-7.68116274070127E-11"
        izz="0.000652901138625573" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Left_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.796078431372549 0.796078431372549 0.796078431372549 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Left_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Left_wheel_Axis"
    type="continuous">
    <origin
      xyz="0 0.212229686537745 0.0183338534091531"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Left_wheel_link" />
    <axis
      xyz="0 1 0" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="Back_MDW_frame_link">
    <inertial>
      <origin
        xyz="-0.00475029171679112 -0.0159989183073881 0.0002537425043838"
        rpy="0 0 0" />
      <mass
        value="0.0181670542093644" />
      <inertia
        ixx="5.72579167306931E-06"
        ixy="-7.35212482813745E-07"
        ixz="3.8141159495066E-08"
        iyy="6.85382817075558E-06"
        iyz="6.10228259069868E-08"
        izz="6.26356011268607E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Back_MDW_frame_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Back_MDW_frame_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Back_MDW_frame_Axis"
    type="continuous">
    <origin
      xyz="-0.165000000000002 0 0.0429999999999998"
      rpy="1.5707963267949 0 -0.0478770767270203" />
    <parent
      link="base_link" />
    <child
      link="Back_MDW_frame_link" />
    <axis
      xyz="0 1 0" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="Back_MDW_wheel_link">
    <inertial>
      <origin
        xyz="2.19269047363468E-15 0 -4.9960036108132E-16"
        rpy="0 0 0" />
      <mass
        value="0.0803777244026511" />
      <inertia
        ixx="2.90627993874311E-05"
        ixy="9.93120762310388E-22"
        ixz="1.82015378594515E-06"
        iyy="2.89117363434887E-05"
        iyz="7.47994503296913E-22"
        izz="5.08427111075994E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Back_MDW_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Back_MDW_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Back_MDW_wheel_Axis"
    type="continuous">
    <origin
      xyz="-0.0186165206066295 -0.0552708014281127 0.0019464470498146"
      rpy="0 0 0" />
    <parent
      link="Back_MDW_frame_link" />
    <child
      link="Back_MDW_wheel_link" />
    <axis
      xyz="-0.0827102895528008 0 -0.996573634009094" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="Front_MDW_frame_link">
    <inertial>
      <origin
        xyz="0.00248501109838795 -0.0159989183073879 0.00405639944951833"
        rpy="0 0 0" />
      <mass
        value="0.0181670542093644" />
      <inertia
        ixx="6.10969442173415E-06"
        ixy="4.0373691414436E-07"
        ixz="-2.46016685875738E-07"
        iyy="6.85382817075558E-06"
        iyz="6.1746067431377E-07"
        izz="5.87965736402121E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Front_MDW_frame_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Front_MDW_frame_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Front_MDW_frame_Axis"
    type="continuous">
    <origin
      xyz="0.165 0 0.0429999999999998"
      rpy="1.5707963267949 0 -0.0478770767270203" />
    <parent
      link="base_link" />
    <child
      link="Front_MDW_frame_link" />
    <axis
      xyz="0 1 0" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="Front_MDW_wheel_link">
    <inertial>
      <origin
        xyz="8.32667268468867E-17 0 -4.16333634234434E-16"
        rpy="0 0 0" />
      <mass
        value="0.0803777244026511" />
      <inertia
        ixx="4.43804269714061E-05"
        ixy="-1.791097142141E-21"
        ixz="-1.01143374248899E-05"
        iyy="2.89117363434887E-05"
        iyz="1.67995338632247E-21"
        izz="3.55250835236243E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Front_MDW_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Front_MDW_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Front_MDW_wheel_Axis"
    type="continuous">
    <origin
      xyz="0.0105759971863268 -0.0552708014281127 0.0154438265701736"
      rpy="0 0 0" />
    <parent
      link="Front_MDW_frame_link" />
    <child
      link="Front_MDW_wheel_link" />
    <axis
      xyz="-0.83696481718083 0 0.547256699183719" />
    <limit
      effort="0"
      velocity="100" />
    <dynamics
      damping="0.1"
      friction="0.1" />
  </joint>
  <link
    name="IMU_link">
    <inertial>
      <origin
        xyz="9.133E-05 -0.0017303 -0.00073244"
        rpy="0 0 0" />
      <mass
        value="0.00064128" />
      <inertia
        ixx="1.1862E-08"
        ixy="1.3728E-14"
        ixz="1.2576E-13"
        iyy="4.4714E-08"
        iyz="7.9871E-13"
        izz="3.3118E-08" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/IMU_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/IMU_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="IMU_link"
    type="fixed">
    <origin
      xyz="-0.00486608888305493 -0.0104697509145408 0.129"
      rpy="1.5707963267949 0 -0.0478770767270203" />
    <parent
      link="base_link" />
    <child
      link="IMU_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="Lidar_link">
    <inertial>
      <origin
        xyz="-6.7849E-06 -0.0040159 -0.00020001"
        rpy="0 0 0" />
      <mass
        value="0.0094944" />
      <inertia
        ixx="3.8141E-06"
        ixy="-4.9853E-11"
        ixz="7.157E-11"
        iyy="4.9345E-06"
        iyz="-8.0756E-10"
        izz="3.8153E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Lidar_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.082353 0.082353 0.082353 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_export_assem#3.SLDASM/meshes/Lidar_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Lidar_link"
    type="continuous">
    <origin
      xyz="0.200500000002106 0 0.18466154484571"
      rpy="1.5707963267949 0 -0.0478770767270203" />
    <parent
      link="base_link" />
    <child
      link="Lidar_link" />
    <axis
      xyz="0 1 0" />
  </joint>
</robot>